### ISSUE TITLE
🐛 File not found in `/tmp/resting_preproc_sub-{sub}_ses-{ses}/func_gather_0/_scan_{scan}/selectrest`

### ISSUE THREAD

**Issue by shnizzedy** [FCP-INDI/app-C-PAC-brainlife.io#2]
**Describe the bug**

```Python
Traceback (most recent call last):
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/plugins/multiproc.py", line 69, in run_node
    result['result'] = node.run(updatehash=updatehash)
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 482, in run
    self, report_type='postexec', is_mapnode=isinstance(self, MapNode))
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/utils.py", line 156, in write_report
    result = node.result  # Locally cache result
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 195, in result
    return _load_resultfile(self.output_dir(), self.name)[0]
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/utils.py", line 364, in load_resultfile
    **modify_paths(outputs, relative=False, basedir=path))
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/utils.py", line 482, in modify_paths
    val, relative=relative, basedir=basedir)
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/utils.py", line 502, in modify_paths
    raise IOError('File %s not found' % out)
OSError: File /tmp/resting_preproc_sub-9002_ses-1/func_gather_0/_scan_rest/selectrest/bids/sub-9002/ses-1/func/sub-9002_ses-1_task-rest_bold.nii.gz not found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.7/concurrent/futures/process.py", line 239, in _process_worker
    r = call_item.fn(*call_item.args, **call_item.kwargs)
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/plugins/multiproc.py", line 72, in run_node
    result['result'] = node.result
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 195, in result
    return _load_resultfile(self.output_dir(), self.name)[0]
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/utils.py", line 364, in load_resultfile
    **modify_paths(outputs, relative=False, basedir=path))
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/utils.py", line 482, in modify_paths
    val, relative=relative, basedir=basedir)
  File "/home/jon/.local/lib/python3.7/site-packages/nipype/pipeline/engine/utils.py", line 502, in modify_paths
    raise IOError('File %s not found' % out)
OSError: File /tmp/resting_preproc_sub-9002_ses-1/func_gather_0/_scan_rest/selectrest/bids/sub-9002/ses-1/func/sub-9002_ses-1_task-rest_bold.nii.gz not found
```

**To Reproduce**
1. try to run this repository on Brainlife

OR

1. Clone this repository
2. Locally recreate the Brainlife directory structure
3. Run `./main`

**Expected behavior**

C-PAC runs the default pipeline on the single subject in `bids`.

**Versions** 
 - C-PAC: v1.7.0
 - Container Platform :
    - Singularity: 2.5.2-HEAD.b258b6514

**Additional context**

Brainlife uses [bl2bids](https://github.com/brainlife/abcd-spec/blob/master/hooks/bl2bids) to create a local directory called `bids` that looks like this
```tree
├── bids
│   ├── dataset_description.json
│   └── sub-9002
│       └── ses-1
│           ├── anat
│           │   ├── sub-9002_ses-1_T1w.json
│           │   └── sub-9002_ses-1_T1w.nii.gz -> ../../../../../5f185b023f121164d2bf5080/5d797315b0578272dd90df95/t1.nii.gz
│           └── func
│               ├── sub-9002_ses-1_task-rest_bold.json
│               └── sub-9002_ses-1_task-rest_bold.nii.gz -> ../../../../../5f185b023f121164d2bf5080/5e546c4b5b9d90232672dae5/bold.nii.gz
```

**Comment by shnizzedy** [comment]
Possibly resolved in 2dc608c; [testing now](https://brainlife.io/project/5f299e1fbeafe973f362a972/process/5f8880e7c7adcba36813d100)

**Comment by shnizzedy** [comment]
Resolved in 2dc608c
